### ISSUE TITLE
Evaluate miraheze.org

### ISSUE THREAD

**Issue by kelson42** [kiwix/web#110]
This might be interesting to see if we could collaborate with Miraheze to be able to provie a full toolchain from content creation to offline usage.

**Comment by Popolechien** [comment]
@JohnFLewis ?

**Comment by JohnFLewis** [comment]
Hi,

Taking a look around this is very much something we can potentially do - and we'd love to see if it can go anywhere.

**Comment by LeBern** [comment]
I would really like to be made aware of whatever happen with that project :-) @JohnFLewis

**Comment by kelson42** [comment]
@LeBern First thing tondo IMO, would be to try to scrape one of the wikis hosted by miraheze.org. Wouldnyou have a preference?

**Comment by LeBern** [comment]
What about All the tropes ? https://allthetropes.org/wiki/

**Comment by Popolechien** [comment]
@LeBern Different website -> please open a separate ticket (you should see a "new issue" button near the top right of this screen)

**Comment by kelson42** [comment]
@LeBern This is hosted on Miraheze?

**Comment by OwenBaines** [comment]
https://allthetropes.miraheze.org/ is the internal name for the wiki.

**Comment by LeBern** [comment]
Well, I think it is : https://allthetropes.org/wiki/All_The_Tropes:History
Since 2015 if I'm correct

**Comment by kelson42** [comment]
Have created a recipe here https://farm.openzim.org/recipes/all_the_tropes

**Comment by labster** [comment]
If you need any settings changed on allthetropes.org, @Arcane21 or myself can approve it from the community point of view.  We'd definitely be happy to see an offline version of our wiki.

**Comment by kelson42** [comment]
@labster Thank you for your offer! Would that be possible to activate Parsoid/VE on the "All the Tropes" wiki?

**Comment by labster** [comment]
Parsoid and VE is enabled, but you need to click to turn it on as a beta feature.  At least, I did.

**Comment by kelson42** [comment]
@JohnFLewis @labster @LeBern We have finally a ZIM of the "All the Tropes" wiki, see https://farm.openzim.org/recipes/all_the_tropes. I have created a tag "miraheze" in the zimfarm so we can identify your wikis easily.

To my opinion, at this stage:
* we can say that we should/would be able to easily create ZIM of any of the Miraheze wikis.
* we should setup a call if we want to deepen a the collaboration and better benefit of each other expertise.

**Comment by kelson42** [comment]
No feedback here, I suspect for the moment the conditions are not gathered for a closer partnership. At least we know it works technically. Let close this ticket for the moment an retry in a while.

**Comment by JohnFLewis** [comment]
@kelson42 I've just stumbled upon this again after a while, I'd be interested in re-opening discussion over this from a Miraheze-wide view point. Let me know what you think.

**Comment by kelson42** [comment]
@JohnFLewis Thank you for coming back to us. Would you be available at some point for a short call on this topic? I would like to better understand how you work and how we could setup something smartly. You can join me per email: kelson ^at^ kiwix.org.

**Comment by JohnFLewis** [comment]
I'll arrange something in the next day or two with you via email.

**Comment by kelson42** [comment]
Miraheze is winding down unfortunately

**Comment by labster** [comment]
No, Miraheze is not winding down.  We just had some drama and now we're doing some reorganization.  I will be serving as Executive Director soon.

Also hi @kelson42, we haven't talked in a while and I hope you're doing well.

**Comment by kelson42** [comment]
@labster Hi, nice feedback and great news :) Reopening the ticket then.

**Comment by rgaudin** [comment]
Wrong repo. @kelson42 reopen where it should be if that's not a dead topic.
